regression/scripts/Vehicles/Cessna_172.py

```diff
@@ -25,77 +25,77 @@ def vehicle_setup():
     # ------------------------------------------------------------------
     #   Initialize the Vehicle
     # ------------------------------------------------------------------        
-    vehicle     = SUAVE.Vehicle()
-    vehicle.tag = 'Cessna_172_SP'
-
+    vehicle                                     = SUAVE.Vehicle()
+    vehicle.tag                                 = 'Cessna_172_SP'
+                                                
     # ------------------------------------------------------------------
     #   Vehicle-level Properties
     # ------------------------------------------------------------------    
 
     # mass properties
-    vehicle.mass_properties.max_takeoff   = 3000. * Units.pounds
-    vehicle.mass_properties.takeoff       = 3000. * Units.pounds
-    vehicle.mass_properties.max_zero_fuel = 3000. * Units.pounds
-    vehicle.mass_properties.cargo         = 0. 
-
-    # envelope properties
-    vehicle.envelope.ultimate_load = 5.7
-    vehicle.envelope.limit_load    = 3.8
-
-    cruise_speed                     = 140. *Units['mph']
-    altitude                         = 13500. * Units.ft
-    atmo                             = SUAVE.Analyses.Atmospheric.US_Standard_1976()
-    freestream                       = atmo.compute_values (0.)
-    freestream0                      = atmo.compute_values (altitude)
-    mach_number                      = (cruise_speed/freestream.speed_of_sound)[0][0] 
-    vehicle.design_dynamic_pressure  = ( .5 *freestream0.density*(cruise_speed*cruise_speed))[0][0]
-    vehicle.design_mach_number       =  mach_number
-    
-    # basic parameters
-    vehicle.reference_area           = 174. * Units.feet**2       
-    vehicle.passengers               = 4
+    vehicle.mass_properties.max_takeoff         = 2550. * Units.pounds
+    vehicle.mass_properties.takeoff             = 2550. * Units.pounds
+    vehicle.mass_properties.max_zero_fuel       = 2550. * Units.pounds
+    vehicle.mass_properties.cargo               = 0. 
+                                               
+    # envelope properties                       
+    vehicle.envelope.ultimate_load              = 5.7
+    vehicle.envelope.limit_load                 = 3.8
+                                                
+    cruise_speed                                = 140. *Units['mph']
+    altitude                                    = 13500. * Units.ft
```

Comment:
this is a bit high

Reply:
I changed to a more reasonable cruise altitude. Originally it was on the upper limit of cruise altitude

Reply:
I wanted you to think a little bit more about how that altitude was used. This altitude is used for the vehicle design dynamic pressure. For this kind of aircraft, that is unlikely to occur at the cruise or service ceiling. Rather it's probably sea level (126 Knots/145 MPH). If you need numbers on the [C172 check out the PIM.](http://www.gaceflyingclub.com/Member%20Download/172S%20Skyhawk%20Information%20Manual%20Searchable.pdf)

I realistically doubt it will change the results. I'm just being nitpicky because some asked for an example Cessna test script on the forum.

Reply:
No worried. I changed it to 145 mph. p.s. I did not add these values, rather the modifications were from reorganizing the scripts (see develop branch). Nevertheless, I changed it to 124 kts which is the specified cruise (75% power) at 8500 ft.